docs/components/dartpad/tabs/scollable_theme/main.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final tabs = [
+      'Tab 1',
+      'Tab 2',
+      'Tab 3',
+      'Tab 4',
+      'Tab 5',
+      'Tab 6',
+      'Tab 7',
+      'Tab 8',
+      'Tab 9',
+    ];
+
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: _buildShrineTheme(),
+      home: DefaultTabController(
+        length: tabs.length,
+        child: Scaffold(
+          appBar: AppBar(
+            title: Text('Scrollable Tabs Theme'),
+            automaticallyImplyLeading: false,
+            bottom: TabBar(
```

Comment:
Set the `indicatorColor` to maybe the shrine brown color here?

Reply:
Done